### PR TITLE
Remove oramod and orarep file type handlers from Windows and Linux.

### LINKED ISSUE #13082: Opening .orarep or .oramod files opens the latest OpenRA installation.
(Windows 7) This is a minor inconvenience if you know this but likely a problem for those who don't. So the latest installment of OpenRA be it a release or playtest will be the go-to client for any replay or mod file opened directly from Windows. If you just installed the latest playtest for example, your files will open it and display a message 'Missing Dependencies' and ask you to fully install the correct client version.

For me it's a minor inconvenience but for newer users probably confusing since they likely already have the necessary client installed.

### PR DESCRIPTION
**Pull request by pchote**
This takes us another small step towards supporting parallel engine / self-contained thirdparty mod installs.

It is a shame to remove direct orarep launching, but as discussed earlier I don't see a feasible way for us to maintain support for this when we have multiple parallel engine installs.  This is one of the only downsides of #12554, and it is far outweighed by all the benefits that brings.

Closes #13082.

### CODE REVIEW

**Review by reaperrr — APPROVED** [review]

File: packaging/windows/OpenRA.nsi [hunk 1]
```diff
@@ -213,8 +203,6 @@ Function ${UN}Clean
 	RMDir /r $INSTDIR\Support
 
 	DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\OpenRA"
-	DeleteRegKey HKLM "Software\Classes\.orarep"
-	DeleteRegKey HKLM "Software\Classes\OpenRA_replay"
```
Comment (abcdefg30) [inline packaging/windows/OpenRA.nsi]: As discussed on IRC: This wants a cleanup rule.

Comment (pchote) [inline packaging/windows/OpenRA.nsi]: Done.

**Comment by pchote** [comment]
Fixed and rebased.

**Comment by pchote** [comment]
Note that I have tested the linux packages and confirmed that they work as expected.

**Review by abcdefg30 — APPROVED** [review]
Cleanup rule seems to work. And the rest is still fine as well.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/da1c6ce4c2d4fee6759988643bdecbec780a7f02...b2c1b1a7d42c92a6f17174c0bfd6098824717fc8)

We should mention this in the news post for Next+1 to avoid confusion.

**Pull request merged**